### ISSUE TITLE
The Hispaniola nations - Coups still happen after the OFN defeats the PPLN and DPR

### ISSUE THREAD

**Issue by PhoneythePhoneSaver** [TNODevs/TNO-Bug-Tracker#2835]
### Operating System

Windows 11

### HOI4 version

1.13.5

### TNO version

1.5.0b

### Do you have other mods in use?

none

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

AAT

### Files

[error.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/13454768/error.log)

### Describe the bug

In Haiti, after Cantave coups and the OFN beats the PPLN and DPR, Haiti ends up getting couped by some random person that isn't Magloire while the Dominican Republic turns communist even after America lets them go.

### Steps to reproduce / Cause of the issue

1. Set game rules where the ceasefire fails and the legion wins in the DR, the communist putsch succeeds, Cantave coups in Haiti, and they defeat the PPLN through war.
2. Wait

### Expected behaviour

In Haiti, Cantave should either hold power or be couped by Magloire in 1967, and whoever America put in power in the Dominican Republic (Bosch or Bonelly) should stay in power

### Screenshots

![image (27)](https://github.com/TNODevs/TNO-Bug-Tracker/assets/143363540/5b191247-fc89-40d7-a8e4-01ebd3dbd794)

**Comment by runelead00** [comment]
Dominican Republic side of issue fixed in dev build

**Comment by runelead00** [comment]
Could you provide the game.log from your game, assuming you haven't restarted since this report

**Comment by PhoneythePhoneSaver** [comment]
Idk if it'll help much but here
[game.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/13455193/game.log)

**Comment by runelead00** [comment]
yeah that one isn't much use, since you just restarted the game before uploading it

**Comment by PhoneythePhoneSaver** [comment]
I'll try it again later then send the log file

**Comment by PhoneythePhoneSaver** [comment]
Okay so I did it again, and although Cantave wasn't replaced by a different leader, the "Insurgency Eradicated in Haiti" event still pops up twice, here's the log (there was another bug that happened to Germany during this run which I'll talk about in another issue)
[game.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/13554725/game.log)
